### PR TITLE
E/speaker info page

### PR DESCRIPTION
**Pull request by wordshaker**
### Requirements

* Page on Speaker Information

### To Run

* To run on localhost:3000 run  `npm run dev`

### CODE REVIEW

File: pages/speaker-information.js [hunk 1]
```diff
@@ -80,58 +103,137 @@ export default () => (
...
             <a name="we-need-to-know"/>
-            <h4>Is there sound/video in your presentation?</h4>
+            <h2>What we need to know</h2>
+            <li>Is there sound/video in your presentation?</li>
+
+            <a name="what-to-prepare"/>
+            <h2>What to prepare</h2>
+            <li>A copy of your presentation on a usb stick</li>
+            <li>A clicker</li>
+            <li>If possible, get any electricals you will need on the day PAT Testeds</li>
```
Comment (Brunty) [inline pages/speaker-information.js]: `Testeds` -> `Tested`

File: pages/speaker-information.js [hunk 2]
```diff
@@ -80,58 +103,137 @@ export default () => (
...
             <a name="room-arrival"/>
-
+            <h2>Arriving at the room</h2>
+            
+            <h4>Please arrive 10 - 15 minutes before your presentation.</h4>
+            <p>
+                This is so that there is enough time to get you all set up.
+            </p>
+            
+            <h4>Introduce yourself to the crew member looking after your roomr</h4>
```
Comment (Brunty) [inline pages/speaker-information.js]: `roomr` -> `room`

File: pages/speaker-information.js [hunk 3]
```diff
@@ -80,58 +103,137 @@ export default () => (
...
+            <p>
+                The crew member will make sure there is water, and will be happy to help you with your presentation
+                needs. They will also help us make sure everything is running smoothly in your room. Please let them know 
+                if there is anything we can provide to help you.
+            </p>
+            
+            <h4>Meet the AV technician and get Miced up</h4>
```
Comment (Brunty) [inline pages/speaker-information.js]: Unsure if it's meant to be `Miced`, `Miked` or `Mic'd`

File: pages/speaker-information.js [hunk 4]
```diff
@@ -80,58 +103,137 @@ export default () => (
...
+            <li>Clicker</li>
+            <li>Backup of slides</li>
+            <li>Laptop charger</li>
+            <li>Timer</li>
+            <li>The awesome <a href="https://open.spotify.com/playlist/0kTPxCiWN0kyYa8FSWpdi2?" target="_blank">Get Pumped Playlist</a></li>
+
+            <h4>Our favourite available list</h4>
+            <p>
+                <a href="https://csswizardry.com/2016/06/speakers-checklist-before-and-after-your-talk/" target="_blank">This list by csswizardry</a> 
+                is often used by us at other events. Thank you Brunty for sharing this with us.
```
Comment (Brunty) [inline pages/speaker-information.js]: Can has link to twitter maybe? :) https://twitter.com/Brunty

File: pages/speaker-information.js (hunk 1 above)
Comment (wordshaker) [reply pages/speaker-information.js]: Hahaha! good spot!

File: pages/speaker-information.js (hunk 2 above)
Comment (wordshaker) [reply pages/speaker-information.js]: I really can't spell!

File: pages/speaker-information.js (hunk 3 above)
Comment (wordshaker) [reply pages/speaker-information.js]: probably the last one? I'll change it

File: pages/speaker-information.js (hunk 4 above)
Comment (wordshaker) [reply pages/speaker-information.js]: Haha! Definitely!

File: pages/speaker-information.js [hunk 5]
```diff
@@ -80,58 +103,137 @@ export default () => (
...
+            <li>Clicker</li>
+            <li>Backup of slides</li>
+            <li>Laptop charger</li>
+            <li>Timer</li>
+            <li>The awesome <a href="https://open.spotify.com/playlist/0kTPxCiWN0kyYa8FSWpdi2?" target="_blank">Get Pumped Playlist</a></li>
+
+            <h4>Our favourite available list</h4>
+            <p>
+                <a href="https://csswizardry.com/2016/06/speakers-checklist-before-and-after-your-talk/" target="_blank">This list by csswizardry</a> 
+                is often used by us at other events. Thank you <a href=" https://twitter.com/Brunty" target="_blank">Brunty</a> for sharing this with us.
```
Comment (Brunty) [inline pages/speaker-information.js]: There's an extra space at the start of the `href=" https"` attribute here

Comment (Brunty) [reply pages/speaker-information.js]: Not sure if it'll cause issues, but who knows with HTML and browsers! :D

**Pull request merged**